app/src/main/java/com/keylesspalace/tusky/network/MastodonApi.java

```diff
@@ -33,6 +31,7 @@
 
 import java.util.List;
 
+import androidx.annotation.Nullable;
```

Comment:
Please remove unused imports in few places

Reply:
I think there's no unused imports in this PR.
Am I missing something?

Reply:
Maybe I'm reading incorrectly on Github, sorry